src/main/java/hudson/plugins/ccm/parser/CcmParser.java

```diff
@@ -141,4 +134,43 @@ public CcmParser(String defaultEncoding) {
 		return annotations;
 	}
 
+	private Priority calculateMetricPriority(Metric metric){
+		Priority priority;
+		if (isMetricHighPriority(metric)) {
+			priority = Priority.HIGH;
+		} else if (isMetricModeratePriority(metric)) {
+			priority = Priority.NORMAL;
+		} else {
+			priority = Priority.LOW;
+		}
+
+		return priority;
+	}
+
+	private boolean isMetricHighPriority(Metric metric){
+		String metricClassification = metric.getClassification();
+		if (metricClassification.contains("high")){
+			return true;
+		}
+		else if (metricClassification.contentEquals("C") || metricClassification.contentEquals("D") ||
```

Comment:
Sure! I ran the Radon complexity tool against a Python module called PynamoDB and updated the PR. You can see the resulting output here https://github.com/jenkinsci/ccm-plugin/blob/a5d795186e747557c54dcd131739bbfc308aa11e/src/test/resources/hudson/plugins/ccm/parser/pynamodb_ccm.xml